hack/make-rules/build_with_container.sh

```diff
@@ -23,11 +23,12 @@ set -o pipefail
 KUBEEDGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
 MOUNTPATH="${MOUNTPATH:-/kubeedge}"
 KUBEEDGE_BUILD_IMAGE=${KUBEEDGE_BUILD_IMAGE:-"kubeedge/build-tools"}
-DOCKER_GID="${DOCKER_GID:-$(grep '^docker:' /etc/group | cut -f3 -d:)}"
+UID_GID="$UID"
```

Comment:
UID="$UID" ?

Reply:
![image](https://user-images.githubusercontent.com/20016835/170998904-f8c58d30-d4aa-43b6-8480-4ac8cddeb657.png)

By default, we use uid as the default parameter for `-u`. Then append the docker's gid to the uid in the form of `uid:gid` if it is present.